java/code/src/com/redhat/rhn/frontend/dto/ErrataOverview.java

```diff
@@ -139,6 +142,30 @@ public void addPackageName(String name) {
     public List getPackageNames() {
         return packageNames;
     }
+    /**
+     * Adds a name to packageNames list.
+     * @param idIn The id to add.
+     */
+    public void addPackageId(long idIn) {
+        pids.add(idIn);
+    }
+    /**
+     * @return Returns the package ids.
+     */
+    public List<Long> getPackageIds() {
+        return pids;
+    }
+
+    /**
+     * @return Package names by ids.
+     */
+    public Map<Long, String> getPackageInfo() {
+        Map<Long, String> m = new HashMap<>();
+        for (int i = 0; i < packageNames.size(); i++) {
+            m.put(pids.get(i), packageNames.get(i));
+        }
+        return m;
+    }
```

Comment:
@lucidd this looks unused.

Reply:
ah right looks like i stopped using it at some point